packages/neuron-ui/src/locales/zh.json

```diff
@@ -1253,7 +1253,7 @@
     },
     "set-start-block-number": {
       "title": "设置同步起始区块",
-      "warn": "请注意，同步起始区块之前的交易不会被同步！",
+      "warn": "为确保数据同步准确性，请输入您钱包第一笔交易的区块编号",
```

Comment:
I think `区块编号` should be `区块高度` because `block hash` could be considered as `编号` too. `区块高度` is more accurate.
Same suggestion for updates in `zh-tw.json`